test/playback.test.js

```diff
@@ -232,101 +236,101 @@ QUnit[testFn]('Big Buck Bunny audio only, groups & renditions same uri', functio
-    type: 'application/x-mpegURL'
+    src: 'https://livesim.dashif.org/livesim/mup_30/testpic_2s/Manifest.mpd',
+    type: 'application/dash+xml'
   });
 });
 
-QUnit[testFn]('Big Buck Bunny Demuxed av, audio only rendition same as group', function(assert) {
-  const done = assert.async();
+if (!videojs.browser.IS_FIREFOX) {
+  QUnit[testFn]('Big Buck Bunny audio only, groups & renditions same uri', function(assert) {
+    const done = assert.async();
 
-  assert.expect(2);
-  const player = this.player;
+    assert.expect(2);
+    const player = this.player;
 
-  playFor(player, 25, function() {
-    assert.ok(true, 'played for at least 25 seconds');
-    assert.equal(player.error(), null, 'has no player errors');
+    playFor(player, 2, function() {
+      assert.ok(true, 'played for at least two seconds');
+      assert.equal(player.error(), null, 'has no player errors');
 
-    done();
-  });
+      done();
+    });
 
-  player.src({
-    src: 'https://d2zihajmogu5jn.cloudfront.net/demuxed-ts-with-audio-only-rendition/master.m3u8',
-    type: 'application/x-mpegURL'
+    player.src({
+      src: 'https://d2zihajmogu5jn.cloudfront.net/audio-only-dupe-groups/prog_index.m3u8',
+      type: 'application/x-mpegURL'
+    });
   });
-});
 
-QUnit[testFn]('Live DASH', function(assert) {
-  const done = assert.async();
+  QUnit[testFn]('Big Buck Bunny Demuxed av, audio only rendition same as group', function(assert) {
+    const done = assert.async();
 
-  assert.expect(2);
-  const player = this.player;
+    assert.expect(2);
+    const player = this.player;
 
-  playFor(player, 2, function() {
-    assert.ok(true, 'played for at least two seconds');
-    assert.equal(player.error(), null, 'has no player errors');
+    playFor(player, 25, function() {
+      assert.ok(true, 'played for at least 25 seconds');
+      assert.equal(player.error(), null, 'has no player errors');
 
-    done();
-  });
+      done();
+    });
 
-  player.src({
-    src: 'https://livesim.dashif.org/livesim/mup_30/testpic_2s/Manifest.mpd',
-    type: 'application/dash+xml'
+    player.src({
+      src: 'https://d2zihajmogu5jn.cloudfront.net/demuxed-ts-with-audio-only-rendition/master.m3u8',
+      type: 'application/x-mpegURL'
+    });
   });
-});
 
-QUnit[testFn]('DASH sidx', function(assert) {
-  const done = assert.async();
-  const player = this.player;
+  QUnit[testFn]('DASH sidx', function(assert) {
+    const done = assert.async();
+    const player = this.player;
 
-  playFor(player, 2, function() {
-    assert.ok(true, 'played for at least two seconds');
-    assert.equal(player.error(), null, 'no errors');
+    playFor(player, 2, function() {
+      assert.ok(true, 'played for at least two seconds');
+      assert.equal(player.error(), null, 'no errors');
 
-    player.one('ended', () => {
-      assert.ok(true, 'triggered ended event');
-      done();
-    });
+      player.one('ended', () => {
```

Comment:
All tests still exist here, some were just moved down to be a part of the block that skips tests on firefox.